haskell/private/cc_wrapper.py.tpl

```diff
@@ -80,8 +80,9 @@ import sys
 import tempfile
 
 WORKSPACE = "{:workspace:}"
-CC = "{:cc:}"
-PLATFORM = "{:platform:}"
+CC = os.environ.get("CC_WRAPPER_CC_PATH", "{:cc:}")
+PLATFORM = os.environ.get("CC_WRAPPER_PLATFORM", "{:platform:}")
```

Comment:
I'll have to split this PLATFORM envvar in BUILD_PLATFORM and TARGET_PLATFORM. The script is using it to address concerns of both.